misk/src/main/kotlin/misk/web/resources/StaticResourceEntry.kt

```diff
@@ -10,8 +10,8 @@ import misk.web.dashboard.ValidWebEntry
  * multibind<StaticResourceEntry>().toInstance(StaticResourceEntry(...))
  * ```
  */
-class StaticResourceEntry(
-  url_path_prefix: String = "/",
+data class StaticResourceEntry(
```

Comment:
Moving these to `data class` make for easier debugging given built-in `toString`, `equals`...etc